exporter/azuremonitorexporter/AUTHENTICATION.md

```diff
@@ -2,42 +2,41 @@
 
 ## Local Authentication
 
-The default authentication mechanism used by the Azure Monitor Exporter is "Local Authentication" - based solely on the Application Insights Instrumentation Key. An example of the exporters section of the configuration is
+The default authentication mechanism used by the Azure Monitor Exporter is "Local Authentication", which relies exclusively on the `InstrumentationKey` obtained from the connection string of the Application Insights. Below is an illustrative example of the exporters section in a configuration file:
 
 ```yaml
 exporters:
    azuremonitor:
-      instrumentation_key: 00000000-0000-0000-0000-000000000000
+      connection_string: "InstrumentationKey=00000000-0000-0000-0000-000000000000;IngestionEndpoint=https://ingestion.azuremonitor.com/"
 ```
 
-Where the zero guid is replaced with the instrumentation key of your Application Insights instance.
+Use the connection string from your Application Insights instance.
 
 The same can be achieved by using an environment variable to hold the key
 
 ```yaml
 exporters:
    azuremonitor:
-      instrumentation_key: ${env:APP_INSIGHTS_KEY}
+      connection_string: ${env:APPLICATIONINSIGHTS_CONNECTION_STRING}
 ```
 
 ## AAD/Entra Authentication
 
-Local Authentication can be disabled in [Application Insights](https://learn.microsoft.com/en-us/azure/azure-monitor/app/azure-ad-authentication) and an AAD based identity can be used in conjunction with the instrumentation key. The instrumentation key is still required by the Exporter, but it is no long  sufficient to just have this in order to import data.
+Local Authentication can be disabled in [Application Insights](https://learn.microsoft.com/en-us/azure/azure-monitor/app/azure-ad-authentication) and an AAD based identity can be used in conjunction with the instrumentation key. The instrumentation key, while still required by the Exporter in the connection string, but it is no longer sufficient to just have this in order to import data.
```

Comment:
```suggestion
Local Authentication can be disabled in [Application Insights](https://learn.microsoft.com/en-us/azure/azure-monitor/app/azure-ad-authentication) and an AAD based identity can be used in conjunction with the instrumentation key. The instrumentation key, while still required by the Exporter in the connection string, is no longer solely sufficient to be able to import data.
```
The wording in the last sentence is a bit unclear. I'm happy to accept a different wording, but the way it's written now is unclear (to me).

Reply:
I agree, this sentence itself is not needed. It is reiterating the previous statement in a different way.